content/intro-to-storybook/svelte/es/conclusion.md

```diff
@@ -7,15 +7,15 @@ Felicitaciones! Creaste tu primer interfaz de usuario en Storybook. En el camino
 
 [📕 **Repositorio de GitHub: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
-[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.com/)
+[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.app/)
 
 Storybook es una poderosa herramienta para React, React Native, Vue, Angular, Svelte y muchos otros frameworks. Cuenta con una próspera comunidad de desarrolladores y una gran cantidad de complementos. Esta introducción nos lleva superficialmente a lo que es posible. Estamos seguros de que una vez que adoptes Storybook, quedarás impresionado por lo productivo que es crear interfaces de usuario duraderas.
 
 ## Aprende más
 
 Quieres bucear más profundo? Aquí algunos recursos útiles:
 
-- [**Documentación oficial de Storybook**](https://storybook.js.org/basics/introduction/) tiene la documentación del API, links comunitarios y una galería de complementos.
+- [**Documentación oficial de Storybook**](https://storybook.js.org/docs/react/get-started/introduction) tiene la documentación del API, links comunitarios y una galería de complementos.
```

Comment:
@gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/svelte/get-started/introduction" in the "Official Storybook documentation" link.